src/test/assets/studentenwerk/mensa-arcisstr/for-generation/2021-09-13.html

```diff
@@ -510,7 +510,7 @@ <h1 class="c-page__title  u-color-current">
 
 		
 
-				</span><p class="js-schedule-dish-description">Ravioli mit Pilzfüllung in pikanter Kräutersauce <span class="c-schedule__marker  c-schedule__marker--type  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--additional  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--allergen  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen="Ei,Gl,GlW,Kn,Mi,Sl"><span class="u-text-sup">[Ei,Gl,GlW,Kn,Mi,Sl]</span></span></p></dd></dl></li>
+				</span><p class="c-menu-dish__title">Ravioli mit Pilzfüllung in pikanter Kräutersauce <span class="c-schedule__marker  c-schedule__marker--type  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--additional  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--allergen  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen="Ei,Gl,GlW,Kn,Mi,Sl"><span class="u-text-sup">[Ei,Gl,GlW,Kn,Mi,Sl]</span></span></p></dd></dl></li>
```

Comment:
how did you genereate these changes?
Did you re-download those html pages?

If yes, could you bump the timestamp in the filename?
If no, could you re-download newer versions of that page?

Reply:
I originally only replaced one xapth, as that was enough for the other commenter on #251. I now downloaded all pages and fixed the other xpaths.